src/base-command.ts

```diff
@@ -77,10 +77,11 @@ export default abstract class BaseCommand extends Command {
 
     // Support -- input ex. architect exec -- ls -la
     const double_dash_index = argv.indexOf('--');
+    let command: string[] = [];
     if (double_dash_index >= 0) {
-      const command = argv.slice(double_dash_index + 1, argv.length).join(' ');
+      command = argv.slice(double_dash_index + 1, argv.length);
       argv = argv.slice(0, double_dash_index);
-      argv.unshift(command);
+      argv.unshift(command.join(' '));
```

Comment:
Should we still be unshifting? If we do this then maybe we don't need command to be an arg.

Reply:
That's a good q. I did add enforcement for `-- COMMAND` since previously you could omit `--` for a single command like `ls`, but even our examples have `architect exec ... -- ls`. With the forced-double-dash we could remove COMMAND as an arg and remove this logic.

If this is the only command that supports `--` as well, can we remove this BaseCommand.parse entirely? The logic becomes much simpler and we can throw it into a much smaller custom parse for Exec only